src/Symfony/Component/Ldap/Entry.php

```diff
@@ -36,6 +36,18 @@ public function getDn()
     }
 
     /**
+     * Returns whether an attribute exists.
+     *
+     * @param $name string The name of the attribute
+     *
+     * @return bool
+     */
+    public function hasAttribute($name)
+    {
+        return isset($this->attributes[$name]);
```

Comment:
Not sure, but we may need to check whether the value is actually null or not.

Could you try running real-world tests with some empty LDAP attributes, to see whether this behaves correctly or not?